simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -32,7 +32,8 @@ public DropwizardExports(MetricRegistry registry) {
         String name = sanitizeMetricName(dropwizardName);
         MetricFamilySamples.Sample sample = new MetricFamilySamples.Sample(name, new ArrayList<String>(), new ArrayList<String>(),
                 new Long(counter.getCount()).doubleValue());
-        return Arrays.asList(new MetricFamilySamples(name, Type.GAUGE, getHelpMessage(dropwizardName, counter), Arrays.asList(sample)));
+        return Arrays.asList(new MetricFamilySamples(name, Type.GAUGE, getHelpMessage(dropwizardName, counter),
+                new ArrayList<MetricFamilySamples.Sample>(Arrays.asList(sample))));
```

Comment:
Is this the most efficient way to produce a list?

Reply:
Not sure about most efficient, but it is suggested as the solution in the thread I linked.